test/acceptance/tests/controller/controller_namespaces_test.go

```diff
@@ -116,10 +116,10 @@ func TestControllerNamespaces(t *testing.T) {
 					// NOTE: No need to clean up because the namespace will be deleted.
 				})
 
-				// On startup, the controller can take upwards of 6s to perform
+				// On startup, the controller can take upwards of 1m to perform
 				// leader election so we may need to wait a long time for
-				// the reconcile loop to run (hence the 20s timeout here).
-				counter := &retry.Counter{Count: 20, Wait: 1 * time.Second}
+				// the reconcile loop to run (hence the 1m timeout here).
+				counter := &retry.Counter{Count: 60, Wait: 1 * time.Second}
```

Comment:
👍

Reply:
Always the solution 😂

Reply:
retry and retry until you succeed or something right? 😛